deploy/docker/seqr/entrypoint.sh

```diff
@@ -72,6 +72,7 @@ if [ $ENABLE_DATABASE_BACKUPS ]; then
 0 0 * * 0 /usr/local/bin/python /seqr/manage.py update_omim --omim-key $OMIM_KEY >> /proc/1/fd/1 2>&1
 0 0 * * 0 /usr/local/bin/python /seqr/manage.py update_human_phenotype_ontology >> /proc/1/fd/1 2>&1
 0 12 * * 1 /usr/local/bin/python /seqr/manage.py detect_inactive_privileged_users >> /proc/1/fd/1 2>&1
+0 2 * * * /usr/local/bin/python /seqr/manage.py check_bam_cram_paths >> /var/log/cron.log 2>&1
```

Comment:
I *just* switched piping the output from the cron jobs to `/proc/1/fd/1` instead of `/var/log/cron.log` as that will allow these logs to get picked up by stackdriver so I can get alerted on failures and check that things are running as expected etc.